### PR TITLE
docs: Describe contract nonce. Changed title to Contract classes and instances in contract-classes.adoc

### PR DESCRIPTION
**Pull request by stoobie**
### Description of the Changes

Changed title to **Contract classes and instances** in contract-classes.adoc, and addresses #1098.

### PR Preview URL

[Contract classes and instances](https://starknet-io.github.io/starknet-docs/pr-1099/documentation/architecture_and_concepts/Smart_Contracts/contract-classes/)

### Check List

- [x] Changes have been done against main branch, and PR does not conflict
- [x] PR title follows the convention: `<docs/feat/fix/chore>(optional scope): <description>`, e.g: `fix: minor typos in code`

- - -
This change is [Reviewable](https://reviewable.io/reviews/starknet-io/starknet-docs/1099)

### CODE REVIEW

File: components/Starknet/modules/architecture_and_concepts/pages/Smart_Contracts/contract-classes.adoc [hunk 1]
```diff
@@ -20,7 +20,9 @@ A contract class does not necessarily have a deployed instance in Starknet.
 A contract class does not necessarily require a deployed instance in Starknet.
 ====
 
-== Working with Classes
+#A contract instance has a nonce, the value of which is the number of transactions originating from this address plus 1. For example, when you deploy an account with a `DEPLOY_ACCOUNT` transaction, the nonce of the account contract in the transaction is `0`. After the transaction, until the account contract sends its first transaction, the nonce is `1`.#
```
Comment (odednaor) [inline components/Starknet/modules/architecture_and_concepts/pages/Smart_Contracts/contract-classes.adoc:23]:
```suggestion
#A contract instance has a nonce, the value of which is the number of transactions originating from this address plus 1. For example, when you deploy an account with a `DEPLOY_ACCOUNT` transaction, the nonce of the account contract in the transaction is `0`. After the deploy transaction, until the account contract invokes its next transaction, the nonce is `1`.#
```

**Review by odednaor — COMMENTED** [review]
Made some minor suggestions.

File: components/Starknet/modules/architecture_and_concepts/pages/Smart_Contracts/contract-classes.adoc (hunk 1 above)
Comment (stoobie) [reply components/Starknet/modules/architecture_and_concepts/pages/Smart_Contracts/contract-classes.adoc:23]: @odednaor these are the three transaction types:
image
So what do you mean by
> the deploy transaction
?

> until the account contract invokes its next transaction

`INVOKE` is a transaction type, so do you mean that the contract sends an `INVOKE` transaction? And what does it mean to invoke a transaction? You can send a transaction, you can invoke a function. I don't see how you can invoke a transaction.

Comment (odednaor) [reply components/Starknet/modules/architecture_and_concepts/pages/Smart_Contracts/contract-classes.adoc:23]: I meant the `DEPLOY_ACCOUNT` transaction.

**Review by odednaor — APPROVED** [review]
lgtm

**Pull request merged**